src/blk/BlockDevice.h

```diff
@@ -64,6 +64,14 @@
 #define  WRITE_LIFE_MAX  	1
 #endif
 
+enum struct FAMode {            // to be used in the buffered parameter
```

Comment:
what does "F" stand for? if it's "file". i'd suggest change it to `write_mode_t` or `wr_mode_t`.

Reply:
File Access mode. 
It differenciates between direct and buffered file I/O, and as such it it not perse targetted at `write`
But any other descriptive name is fine by me.

Reply:
i don't think it's specific to "file" even we are using this mode to choose the right *file* descriptor. what we are accessing are block devices.

Reply:
@tchaikov 
Then call ik blk_access_mode_t??